Sources/CoreMethods/Core/Extensions/CoreMethods+CreateToken.swift

```diff
@@ -14,8 +14,8 @@ extension CoreMethods {
         
         return try await tokenization(
```

Comment:
Should you use this function, we advise you to ensure that the card data collection is secure, as you will not be using our PCI fields where we internalize and secure this information on our end.